spec/index.html

```diff
@@ -279,61 +279,127 @@ <h2>Typographical conventions</h2>
-    <dd>A <a>dataset</a>
-      as specified by [[!RDF11-CONCEPTS]].
-      For the purposes of this specification, an <a>RDF dataset</a>
-      is considered to be a set of <a>quads</a></dd>
-    <dt><dfn data-cite="RDF11-CONCEPTS#dfn-blank-node">blank node</dfn></dt>
-    <dd>A <a>blank node</a>
-      as specified by [[!RDF11-CONCEPTS]]. In short, it is a node in a graph that is
-      neither an <a>IRI</a>, nor a
-      <a>literal</a>.</dd>
-    <dt><dfn data-cite="RDF11-CONCEPTS#dfn-blank-node-identifier">blank node identifier</dfn></dt>
-    <dd>A <a>blank node identifier</a>
-      as specified by [[!RDF11-CONCEPTS]]. In short, it is a <a>string</a> that begins
-      with <code>_:</code> that is used as an identifier for a
-      <a>blank node</a>. <a>Blank node identifiers</a>
-      are typically implementation-specific local identifiers; this document
-      specifies an algorithm for deterministically specifying them.</dd>
-    <dd>
-      Concrete syntaxes, like [[Turtle]] or [[N-Quads]], prepend blank node identifiers with the <code>_:</code> string
-      to differentiate them from other nodes in the graph. This affects the 
-      canonicalization algorithm, which is based on calculating a hash over the representations of <a>quads</a> in this format.
-    </dd>
-  </dl>
+  <section id="canon-terms">
+    <h3>Terms defined by this specification</h3>
+    <dl data-sort="ascending">
+      <dt><dfn data-lt="input dataset|input datasets">input dataset</dfn></dt>
+      <dd>The abstract <a>RDF dataset</a> that is provided as input to
+        the algorithm.</dd>
+      <dt><dfn>normalized dataset</dfn></dt>
+      <dd>The immutable, abstract <a>RDF dataset</a> and set of normalized
+        <a>blank node identifiers</a> that are produced as output by the algorithm.
+        A <a>normalized dataset</a> is a restriction on an <a>RDF dataset</a>
+        where all nodes are labeled, and <a>blank nodes</a> are labeled with canonical <a>blank node identifiers</a>
+        consistent with running this algorithm on a base <a>RDF dataset</a>.
+        A concrete serialization of a <a>normalized dataset</a> MUST label
+        all <a>blank nodes</a> using these stable <a>blank node identifiers</a>.</dd>
+      <dt><dfn>identifier issuer</dfn></dt>
+      <dd>An identifier issuer is used to issue new <a>blank node identifier</a>. It
+        maintains a
+        <a href="#bn-issuer-state">blank node identifier issuer state</a>.</dd>
+      <dt><dfn>hash</dfn></dt>
+      <dd>The lowercase, hexadecimal representation of a message digest.</dd>
+      <dt><dfn>hash algorithm</dfn></dt>
+      <dd>The hash algorithm used by <a>URDNA2015</a>, namely, SHA-256.</dd>
+      <dt><dfn data-lt="mention">mention set</dfn></dt>
+      <dd>The set of all <a>quads</a> in a <a>dataset</a>
+        that mention a node <var>n</var> is called its <a>mention set</a> of <var>n</var>,
```

Comment:
the term "mention", though intuitive, is not really defined. Either it should be separately defined, or use here something "The set of all <a>quads</a> in a <a>dataset</a> that has <var>n<var> as one of its components".

("component" is used in, e.g., https://www.w3.org/TR/rdf11-concepts/#section-triples, so it is o.k. to use that term without an extra definition.)

Reply:
Addressed in b6bf3e294447db5b8ba8d94c0cf23706b0997e66.